.github/workflows/auto-approve.yml

```diff
@@ -1,15 +1,55 @@
 # Approve PRs with "pr/auto-approve". mergify takes care of the actual merge.
 
 name: auto-approve
-on: pull_request
+on:
+  pull_request:
+    types:
+      - labeled           # Run when labels are updated
+      - opened            # Run when a PR is initially submitted
+      - ready_for_review  # Run when a Draft PR is upgraded PR
+      - reopened          # Run when a PR is re-opened
+      - synchronize       # Run when a PR head branch is updated
 
 jobs:
   auto-approve:
-    if: contains(github.event.pull_request.labels.*.name, 'pr/auto-approve')
+    if: contains(github.event.pull_request.labels.*.name, 'pr/auto-approve') || github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
```

Comment:
Guess this means we can get rid of the [labeler](https://github.com/aws/jsii/blob/main/.github/workflows/pr-labeler.yml) workflow. Alternatively, we can make the labeler post a comment and leave this workflow as is...your call.